helix-view/src/editor.rs

```diff
@@ -59,6 +59,12 @@ pub struct FilePickerConfig {
     /// WalkBuilder options
     /// Maximum Depth to recurse directories in file picker and global search. Defaults to `None`.
     pub max_depth: Option<usize>,
+    /// File picker width, treated as a factor. Defaults to 0.9.
+    pub width_factor: f32,
+    /// File picker height, treated as a factor. Defaults to 0.9.
+    pub height_factor: f32,
+    /// File picker preview width, from 0.0 to 1.0 treated as a factor, > 1 width in characters. Defaults to 0.5.
```

Comment:
```suggestion
    /// File picker preview width, from 0.0 to 1.0 treated as a factor, > 1 width in one-width characters. Defaults to 0.5.
```
We should mention one-width characters since there are two widths characters. Or we should just say columns.

Reply:
Thank you @pickfire. I have implemented `preview_min_width` related to your previous post (I forgot, sorry) and `file_min_width` that works similar to `preview_max_width` but I think former allows more control and allows to remove hardcoded constant `MIN_SCREEN_WIDTH_FOR_PREVIEW`.
Could you tell me your impressions?